src/components/services/Docuseal.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <Generic :item="item">
+    <template #content>
+      <p class="title is-4">{{ item.name }}</p>
+      <p class="subtitle is-6">
+        <template v-if="item.subtitle">
+          {{ item.subtitle }}
+        </template>
+        <template v-else-if="versionstring">
+          Version {{ versionstring }}
+        </template>
+      </p>
+    </template>
+    <template #indicator>
+      <div v-if="status" class="status" :class="status">
+        {{ status }}
+      </div>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+
+export default {
+  name: "Docuseal",
+  components: {
+    Generic,
+  },
+  mixins: [service],
+  props: {
+    item: Object,
+  },
+  data: () => ({
+    status: null,
+    versionstring: null,
+  }),
+  created() {
+    this.fetchStatus();
+  },
+  methods: {
+    fetchStatus: async function () {
+      this.fetch("/version", null, false)
+        .then((response) => {
+          this.status = "online";
+          this.versionstring = response;
+        })
+        .catch((e) => {
+          this.status = "offline";
```

Comment:
Maybe also set the version to `unknown` or `n/a` ?
```suggestion
          this.status = "offline";
          this.versionstring = "n/a";
```

Reply:
I prefer it to remain `null` if the fetch fails, since that'll show nothing for a subtitle instead of `Version n/a` or `Version unknown`, which doesn't really add value. We know it's offline anyway. What do you think?

I'm fine either way. If you think we should use `n/a` or `unknown`, then we should probably do that for the other custom services that currently leave it as `null` if the fetch fails (Wallabag, Gitea/Forgejo, Nextcloud, Olivetin).